.github/actions/build-evm-client/ethjs/action.yml

```diff
@@ -0,0 +1,36 @@
+name: 'Build EthereumJS monorepo'
+description: 'Builds the EthereumJS monorepo'
+inputs:
+  repo:
+    description: 'Source repository to use to build EthereumJS'
+    required: true
+    default: 'ethereumjs/ethereumjs-monorepo'
+  ref:
+    description: 'Reference to branch, commit, or tag to use to build EthereumJS'
+    required: true
+    default: 't8ntool'
```

Comment:
Yes this is fine! However I will then wait marking this PR ready for review (and thus ready to merge) once we get t8ntool merged (which will take some time, since I need to do a massive cleanup of that PR)

Reply:
Have marked the PR ready for review, should be merged ASAP, will point to master.